src/Client.java

```diff
@@ -0,0 +1,55 @@
+import java.io.IOException;
+import java.net.*;
+
+public class Client {
+    final int PORT = 9876;
+    DatagramSocket clientSocket;
+
+    public Client() {
+        System.out.println("Client()");
+
+        try {
+            clientSocket = new DatagramSocket();
+        } catch (SocketException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public String sendMessageToServer(String msg) {
+        System.out.println("sendMessageToServer()");
+
+        byte[] buf;
+        String receivedSentence = null;
+        InetAddress IPAddress = null;
+        try {
+            IPAddress = InetAddress.getByName("localhost");
```

Comment:
Let's pull "localhost" to a final field as well.

Reply:
okay